nautobot/core/views/mixins.py

```diff
@@ -593,6 +596,7 @@ class ObjectListViewMixin(NautobotViewSetMixin, mixins.ListModelMixin):
         "per_page",  # used by get_paginate_count
         "sort",  # table sorting
     )
+    use_new_ui = True
```

Comment:
Does this result in all of `NautobotUIVIewSet` (e.g. the Circuits views) getting `use_new_ui=True` (even for its create/edit/delete views)? It seems like it would.